### PR TITLE
Update parsing

### LINKED ISSUE #19: Update CLI-input
Switch `sys.argv` to `argparse`

### PR DESCRIPTION
**Pull request by zhong-al**
Resolves #19 
- Switch to using `argparse` to parse arguments more robustly & provide argument hints
- Some formatting from autopep8

### CODE REVIEW

File: player.py [hunk 1]
```diff
@@ -268,15 +284,23 @@ def player(folder, save):
...
+        type=str,
+        help='path to folder with metadata and actions',
+        required=True
+    )
+    local_parser.add_argument(
+        '--save',
+        action='store_true',
+        help='Flag to save video'
+    )
+    return local_parser
```
Comment (egrace479) [inline player.py:301]:
```suggestion
    return local_parser.parse_args()
```

File: player.py [hunk 2]
```diff
@@ -268,15 +284,23 @@ def player(folder, save):
...
-        exit(0)
-    elif len(sys.argv) == 2:
-        folder = sys.argv[1]
-        save = True#False
-    elif len(sys.argv) == 3:
-        folder = sys.argv[1]
-        save = bool(sys.argv[2])
-
-    player(folder, save)
+    args = get_parser().parse_args()
```
Comment (egrace479) [inline player.py:305]:
```suggestion
    args = get_parser()
```

File: tracks_extractor.py [hunk 3]
```diff
@@ -205,19 +221,29 @@ def tracks_extractor(video, annotation, tracking):
...
+        type=str,
+        help='path to folder containing annotations',
+        required=True
+    )
+    local_parser.add_argument(
+        '--tracking',
+        action='store_true',
+        help='Flag to use external tracker instead of CVAT tracks'
+    )
+    return local_parser
```
Comment (egrace479) [inline tracks_extractor.py:244]:
```suggestion
    return local_parser.parse_args()
```

File: tracks_extractor.py [hunk 4]
```diff
@@ -205,19 +221,29 @@ def tracks_extractor(video, annotation, tracking):
...
-        tracking = False
-    # tracking=True: use external tracker instead of CVAT tracks.
-    # tracking=False: use CVAT tracks.
-    elif len(sys.argv) == 4:
-        video = sys.argv[1]
-        annotation = sys.argv[2]
-        tracking = bool(sys.argv[3])
-
-    tracks_extractor(video, annotation, tracking)
+    args = get_parser().parse_args()
```
Comment (egrace479) [inline tracks_extractor.py:248]:
```suggestion
    args = get_parser()
```

**Review by egrace479 — COMMENTED** [review]
One description that is a bit unclear. Otherwise, they're all the same suggestion to return the parsed args from `get_parser`.

File: cvat2ultralytics.py [hunk 5]
```diff
@@ -154,30 +165,51 @@ def cvat2ultralytics(video_path, annotation_path, dataset, skip):
...
+    local_parser.add_argument(
+        '--dataset',
+        type=str,
+        help='path to output dataset files',
+        required=True
+    )
+    local_parser.add_argument(
+        '--skip',
+        type=int,
+        help='process one out of skip frames',
```
[the remark this answers is not in the document]
Comment (zhong-al) [reply cvat2ultralytics.py:207]: from line 142, I believe it skips skip number of frames

[the remark this answers is not in the document]
Comment (egrace479) [reply cvat2ultralytics.py]: Ah, then maybe adjust the description to 'process one out of skip number of frames'?

**Review by egrace479 — APPROVED** [review]
LGTM

**Pull request merged**